content/references/rippled-api/admin-rippled-methods/status-and-debugging-methods/validator_list_sites.md

```diff
@@ -119,11 +119,24 @@ Each member of the `validator_sites` field array is an object with the following
 
 | `Field`                | Type             | Description                     |
 |:-----------------------|:-----------------|:--------------------------------|
-| `last_refresh_status`  | String           | If present, the[`ListDisposition`](https://github.com/ripple/rippled/blob/master/src/ripple/app/misc/ValidatorList.h) of the most recent refresh of the site. If missing, the site has not yet been succesfully queried. |
-| `last_refresh_time`    | String           | Human readable time when the site was last queried. If missing, the site has not yet been succesfully queried. |
+| `last_refresh_status`  | String           | If present, shows the status of the most recent refresh of the site. If missing, the site has not yet been successfully queried. See **Site Status Values** below for possible states and their meanings. |
```

Comment:
"See **Site Status Values** below" - In this case, I'm not sure it's helpful to change this, but some say not to use "below" because of the possibility of other forms of display or reading. If you just link to Site Status Values, then the location is unambiguous, no matter how the position of the text changes or how it is consumed (such as by screenreader).